### ISSUE TITLE
`Communication`: Resolved question stay in the search list with an enabled  "Unresolved" filter 

### ISSUE THREAD

**Issue by JanaNF** [ls1intum/Artemis#9704]
### Describe the bug

When I search for unresolved questions and answer/resolve them, I still see them in the unresolved list. I expected the question to go away (through an automated update) and not appear on the list. I have to reload the whole page so as not to see it. This could be intentional, but I think it would be more practical if it is automatically gone.

### To Reproduce

1. Go to the communication tab
2. Click in the search bar and search for nothing (enter a space into it)
3. Click on the "Unresolved" check box
4. Go to a question and answer in a thread
5. Click on resolved
6. Now, you still see the resolved question.

### Expected behavior

I expected the question to go away and not show up in the search.

### Screenshots

![image](https://github.com/user-attachments/assets/b6dc7c85-4e3b-46df-bf4f-6d3b6f25d3bb)

### Which version of Artemis are you seeing the problem on?

7.6.5

### What browsers are you seeing the problem on?

Chrome

**Comment by cremertim** [comment]
Is this issue still existing @JanaNF, i tried to reproduce it but i was not able to see the bug.

**Comment by cremertim** [comment]
I believe the behavior is working as intended. When resolving a post, the search is not immediately triggered. However, if you perform the search again (for example as step 7), the newly resolved posts no longer appear in the list.

@krusche Could you kindly confirm if this is the expected behavior?

**Comment by krusche** [comment]
Yes, we do not have a live search at the moment. It filters data directly in the database using pagination. We could think about updating the search result for this particular case though, as long as the UI is not jumping or flickering. I agree it would make sense for the use case of going through unresolved messages and resolving them

**Comment by atharvamp** [comment]
This issue is still relevant. The now resolved comment does not go away from the search results.

<img width="429" height="198" alt="Image" src="https://github.com/user-attachments/assets/7bd063f4-e17a-4982-9259-ede9de701842" />

<img width="539" height="144" alt="Image" src="https://github.com/user-attachments/assets/78c936f5-0ca8-4bd7-873f-307be7a3e290" />

**Comment by Jess-hub09** [comment]
I agree with this issue, I’ve noticed the same behavior quite often. When resolving a question from the “Unresolved” filtered list, it still appears until the entire page is reloaded. It would be much more intuitive if the item disappeared automatically after being marked as resolved, similar to other dynamic updates in Artemis. An automatic refresh of the filtered list would improve usability.
